rpi5.yaml

```diff
@@ -0,0 +1,207 @@
+desc: "Raspberry 5 with xen dom0less"
+min_ver: "0.22"
+
+variables:
+  # Common build configuration
+
+  YOCTOS_WORK_DIR: "yocto"
+
+  DOM0_OS: "zephyr"
+
+  # Raspberry 5 build configuration
+
+  MACHINE: "raspberrypi5"
+  SOC_FAMILY: "bcm2712"
```

Comment:
These lines should be removed, because we have added `--MACHINE rpi5` which sets MACHINE and SOC_FAMILY.

Reply:
Removed